flexmeasures/data/scripts/clean_database.sh

```diff
@@ -23,20 +77,26 @@ function create_database() {
 
 # function for deleting the old database
 function delete_database() {
- echo "Deleting database ..."
- sudo -i -u postgres dropdb -U postgres $1
+ echo "Dropping database ..."
+ if sudo -i -u postgres dropdb -U postgres $1; then
+   echo "$1 database is dropped"
+   exit 0
```

Comment:
I believe that within functions (of which you check the result later) you do not want to `exit`, you want to `return`.

This also happens in `create_user`. Please check that functions return, then get checked in the main loop, and then exit there.